db/migrate/20221019131902_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :posts_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] I was unable to set up your app locally because the users table already exists from earlier migrations. Kindly remove this file to fix that.

- [ ] If you want to change the `post_counter` to an `integer` type then you should simply run the `change column` migration, instead of creating a new table just for that. Look at this example: [https://stackoverflow.com/a/67363686](https://stackoverflow.com/a/67363686)